spec.md

```diff
@@ -1689,6 +1700,18 @@ For success responses, the following fields are defined:
 }
 ```
 
+## Corrupt Service Instances
+
+When an update or delete operation fails, the Service Instance MAY be corrupt.
+A corrupt instance MAY be misconfigured, in an invalid state, not reachable, or
+not working at all.
+Platforms SHOULD not try to create bindings for this instance anymore.
+Whether or not a corrupt instance can be repaired by, for example, updating it
+again, is undefined.
+Deprovisioning a corrupt instance SHOULD still be possible. A Platform MUST
+remember the Service Instance until it is successfully deprovisioned or it has
+been cleaned up as an orphan.
```

Comment:
It really feels like we need some text here about the Platform performing orphan mitigation. What if we combine #598 into this PR so we get the text just right.